includes/class-wc-cart.php

```diff
@@ -51,6 +51,13 @@ class WC_Cart extends WC_Legacy_Cart {
 	public $applied_coupons = array();
 
 	/**
+	 * Contains an array of discount objects applied to the cart.
+	 *
+	 * @var array
+	 */
+	protected $manual_discounts = array();
```

Comment:
I like 'cart discounts'! It was confusing as just 'discounts' because there is so much stuff in WC_Cart that already has the word 'discount' in it. We need some sort of differentiation.